### DISCUSSION TITLE
q-table virtual-scroll on grid mode

### DISCUSSION THREAD

**Discussion by palaciosdiego** [quasarframework/quasar#9543] [General - Components / Directives / etc]
Hi, i'm struggling to make a grid q-table to load items on scroll (virtual-scroll), already tried with non-grid table and works but not working with grid type tables. I wonder if any of you have made it, any help will be well received.

Cheers, Diego

**Comment by metalsadman** [comment]
grid is not supported for q-table virtual scroll.
you can workaround something like this but it's not that dynamic as using the item slot. 
a rough example https://codepen.io/metalsadman/pen/BaWrVaM?editors=1010.
